examples/backup_vm.py

```diff
@@ -214,8 +215,15 @@ def cmd_download(args):
 
     connection = common.create_connection(args)
     with closing(connection):
-        download_backup(
-            connection, args.backup_uuid, args, incremental=args.incremental)
+        verify_vm_exists(connection, args.vm_uuid)
```

Comment:
Nothing should come before with closing(). If the call fails, we don't close
the connection.

verify_backup_exists() is still used in stop_backup(), should move to to cmd_stop()
but this can be done later.